grab.py

```diff
@@ -2,7 +2,7 @@
 from typing import Any, Dict, List, Sequence
 
 from kittens.tui.handler import result_handler
-from kitty.typing import BossType
+from kitty.typing_compat import BossType
```

Comment:
This would break the kitten on older versions of Kitty.

Ubuntu `noble` (current LTS) carries Kitty 0.32.2 and Debian `bookworm` has 0.26.5, so please do it in a backward-compatible way, importing from the new name under a `try` block, falling back to the old name on `ImportError`.

Reply:
OK, I will make this change and resubmit.